.github/workflows/sync-to-oss.yml

```
@@ -0,0 +1,314 @@
+
+  trigger-oss-release:
+    name: 🚀 Trigger OSS Release
+    needs: sync-to-oss
+    runs-on: ubuntu-latest
+    if: contains(github.event.head_commit.message, 'release:')
+    
+    steps:
+      - name: 🎯 Trigger release workflow in OSS repo
+        env:
+          GH_TOKEN: ${{ secrets.OSS_SYNC_TOKEN }}
+        run: |
```

actionlint (1.7.7)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)